pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -6254,7 +6254,7 @@ div.dijitDateTextBox.dijitTextBox {
 #DP_range2_lookup,
 div.dijitDateTextBox input.dijitInputInner[type='text'] {
   padding-right: 0px;
-  padding-top: 0px;
+  padding-top: 0px !important;

```

Comment:
there is a new class overwriting this and is quite general, ill check what are the implications of removing that other important